src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -116,7 +116,7 @@ class TaskForm extends React.Component<Props, State> {
         this.clearForm();
     };
 
-    handleDueDateChange = (date: ?Date): void => {
+    handleDueDateChange = (date?: Date): void => {
```

Comment:
Does this clear the state as expected when the user removes the date from the form field?

Reply:
No, it should be a nullable value

Reply:
DatePicker.js cannot take a null value btw per its flow declaration.

Reply:
@priyajeet, that's fine as long as it actually clears the state when using `undefined` rather than `null`. Does the component still continue to work the same way with this change?

Reply:
Diff seems outdated, check the new diff